pmlc/dialect/tile/builder.cc

```diff
@@ -612,6 +612,7 @@ TileBuilder::MakeProgram(StringRef name, const ProgramMutations &mutations,
   // Construct a function to represent the entire program
   auto initialFuncType = FunctionType::get(inputTypes, {}, getContext());
   auto funcOp = FuncOp::create(loc, name, initialFuncType, {});
+  funcOp.setAttr("entrypoint", mlir::UnitAttr::get(getContext()));
```

Comment:
Actually, there's an `entry` member on the `Program` which was meant to contain this information. If we don't need that anymore, we should remove it from `Program`.

Reply:
> Could we make a separate PR for moving/reorganizing the directory structure? It's hard to review the diff to see what has changed in `jit_executable.cc` for instance.

I could use some more guidance on when a PR has "too little meat on its bones" vs. "hard to review".  :-)   In the meantime, could you look at the individual commits?

Reply:
The issue is that when I look at `jit_executable.cc`, it all looks like new code, but it's actually just a renamed file. It'd be good if we could make a NFC commit for renames/reorgs so that I can see the delta to the executable code. Like temporarily define the contents of `jit_exectuable.cc` in its original location (which now I'm not sure where it was, it used to be in `compiler/executable.cc`.